src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -62,6 +63,21 @@ public static Name parseName(String name) throws ParseException {
         return new Name(trimmedName);
     }
 
+    /**
+     * Parses a {@code String name} into a {@code Name}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code name} is invalid.
+     */
+    public static Title parseTitle(String title) throws ParseException {
+        requireNonNull(title);
+        String trimmedTitle = title.trim();
+        if (!Title.isValidTitle(trimmedTitle)) {
+            throw new ParseException(Title.MESSAGE_CONSTRAINTS);
+        }
+        return new Title(trimmedTitle);
```

Comment:
This is a new method that replaces the old `new Title(parseName(name).get())`, since the ParseException from that throws the message constraints from Name rather than Title.